solr/solr-ref-guide/src/the-well-configured-solr-instance.adoc

```diff
@@ -31,6 +31,8 @@ This section covers the following topics:
 
 <<solr-plugins.adoc#solr-plugins,Solr Plugins>>: Introduces Solr plugins with pointers to more information.
 
+<<solr-packages.adoc, Packages in Solr>>: Deploying, installing, updating packages with plugins into Solr Cluster
```

Comment:
@noblepaul This is likely the cause of your pre-commit failure. The link should be formed like `<<solr-packages.adoc#solr-packages,Packages in Solr>>:`. Note the anchor being added.